apps/precursor-chemicals/fields/index.js

```diff
@@ -197,5 +197,63 @@ module.exports = {
     legend: {
       className: 'govuk-!-margin-bottom-6'
     }
+  },
+  'has-anyone-received-criminal-conviction': {
+    isPageHeading: true,
+    mixin: 'radio-group',
+    validate: [ 'required' ],
+    options: [
+      {
+        value: 'yes'
+      },
+      {
+        value: 'no'
+      }
+    ],
+    className: ['govuk-radios', 'govuk-radios--inline'],
+    legend: {
+      className: 'govuk-!-margin-bottom-6'
+    }
+  },
+  'invoicing-address-line-1': {
+    mixin: ['input-text'],
```

Comment:
needs to be changed to `mixin: 'input-text'` everywhere. 
Currently incorrect syntax doesn't affect the field and doesn't throw an error because 'input-text' is the default, but we agreed to specify it anyway for clarity.